services/core-api/app/api/now_applications/resources/now_application_resource.py

```diff
@@ -71,6 +73,17 @@ def put(self, application_guid):
 
         data = request.json
 
+        verified_by_user_date = data.get('verified_by_user_date', None)
+        decision_by_user_date = data.get('decision_by_user_date', None)
+        if can_edit_now_dates():
```

Comment:
Same as prior validation comment

Reply:
The validations for these are all handled on the FE